### ISSUE TITLE
`release/0.16` branch is pulling torch==2.0.1+cu118 for linux GPU jobs

### ISSUE THREAD

**Issue by NicolasHug** [pytorch/vision#7923]
This is very likely the cause of the failures in https://github.com/pytorch/vision/actions/runs/6037283607/job/16384281192 and in https://hud.pytorch.org/pr/pytorch/vision/7921

CC @atalman

**Comment by atalman** [comment]
@pmeier looks like this related to following install command:
```
ltt install --progress-bar=off --pytorch-computation-backend=cuda11.8 --pytorch-channel=test torch
```

executing correct installation works correctly:
```
pip3 install torch torchvision --index-url https://download.pytorch.org/whl/test/cu118
WARNING: Ignoring invalid distribution -orch (/data/home/atalman/aug29_pypi/lib/python3.10/site-packages)
Looking in indexes: https://download.pytorch.org/whl/test/cu118
Collecting torch
  Using cached https://download.pytorch.org/whl/test/cu118/torch-2.1.0%2Bcu118-cp310-cp310-linux_x86_64.whl (2325.9 MB)
Collecting torchvision
  Using cached https://download.pytorch.org/whl/test/cu118/torchvision-0.16.0%2Bcu118-cp310-cp310-linux_x86_64.whl (6.2 MB)
Requirement already satisfied: filelock in ./aug29_pypi/lib/python3.10/site-packages (from torch) (3.12.3)
Requirement already satisfied: typing-extensions in ./aug29_pypi/lib/python3.10/site-packages (from torch) (4.7.1)
Requirement already satisfied: sympy in ./aug29_pypi/lib/python3.10/site-packages (from torch) (1.12)
Requirement already satisfied: networkx in ./aug29_pypi/lib/python3.10/site-packages (from torch) (3.1)
Requirement already satisfied: jinja2 in ./aug29_pypi/lib/python3.10/site-packages (from torch) (3.1.2)
Requirement already satisfied: fsspec in ./aug29_pypi/lib/python3.10/site-packages (from torch) (2023.6.0)
Collecting pytorch-triton==2.1.0+e6216047b8 (from torch)
  Downloading https://download.pytorch.org/whl/test/pytorch_triton-2.1.0%2Be6216047b8-cp310-cp310-linux_x86_64.whl (96.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 96.4/96.4 MB 19.2 MB/s eta 0:00:00
Requirement already satisfied: numpy in ./aug29_pypi/lib/python3.10/site-packages (from torchvision) (1.25.2)
Requirement already satisfied: requests in ./aug29_pypi/lib/python3.10/site-packages (from torchvision) (2.28.1)
Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in ./aug29_pypi/lib/python3.10/site-packages (from torchvision) (9.3.0)
Requirement already satisfied: MarkupSafe>=2.0 in ./aug29_pypi/lib/python3.10/site-packages (from jinja2->torch) (2.1.3)
Requirement already satisfied: charset-normalizer<3,>=2 in ./aug29_pypi/lib/python3.10/site-packages (from requests->torchvision) (2.1.1)
Requirement already satisfied: idna<4,>=2.5 in ./aug29_pypi/lib/python3.10/site-packages (from requests->torchvision) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./aug29_pypi/lib/python3.10/site-packages (from requests->torchvision) (1.26.13)
Requirement already satisfied: certifi>=2017.4.17 in ./aug29_pypi/lib/python3.10/site-packages (from requests->torchvision) (2022.12.7)
Requirement already satisfied: mpmath>=0.19 in ./aug29_pypi/lib/python3.10/site-packages (from sympy->torch) (1.3.0)
WARNING: Ignoring invalid distribution -orch (/data/home/atalman/aug29_pypi/lib/python3.10/site-packages)
Installing collected packages: pytorch-triton, torch, torchvision
Successfully installed pytorch-triton-2.1.0+e6216047b8 torch-2.1.0+cu118 torchvision-0.16.0+cu118
```

**Comment by pmeier** [comment]
Fixed in #7924.
